core/ir/opnd_api.h

```diff
@@ -1695,6 +1695,11 @@ typedef enum _dr_opnd_flags_t {
     DR_OPND_EXTENDED = 0x20,
     /** This immediate integer operand should be interpreted as an AArch64 extend type. */
     DR_OPND_IS_EXTEND = 0x40,
+    /** Registers with this flag should be considered vectors and have a subsize
+     *  representing their element size. Doubled up with _IS_EXTEND as that can
+     *  only apply to immd values and saves increasing the size of the containing
+     *  structure. */
+    DR_OPND_IS_VECTOR = 0x40,
```

Comment:
This is the same value as DR_OPND_IS_EXTEND.  Is that deliberate b/c the assumption is one is for ints and one for regs?  If so please make that explicit with a (non-doxygen) comment: otherwise it looks like a bug.  But if we have plenty of space I think it would be better to just use a non-overlapping value (don't recall how many bits we have for these flags).